### PR TITLE
Now users can text search :D

### PR DESCRIPTION
**Pull request by TomasGonzalez**
## PR Checklist

Please check all that apply to this PR using "x":

- [ x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number:

## What is the current behavior?
The user is unable to search while using the desktop app.

## What is the new behavior?
If the user presses ctrl-f or command-f while using the desktop app, a floating input field appears that allows the user to search on any page.

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks so much for the PR, we'll get a review on this soon! Can we show you [some appreciation](https://lbry.com/faq/appreciation)?

File: ui/component/searchScreenInput/view.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    };
+
+    window.addEventListener('keydown', handleKeyPress);
+    return () => window.removeEventListener('keydown', handleKeyPress);
+  }, [searchString]);
+
+  return (
+    <input
+      ref={inputRef}
+      onKeyPress={(e) => e.key === 'Enter' && window.find(searchString)}
```
Comment (seanyesmunt) [inline ui/component/searchScreenInput/view.jsx:26]: To match Chromes behavior `window.find` should be called with every key press, not just when you click enter. This code can be combined with `onChange`

**Review by seanyesmunt — CHANGES_REQUESTED** [review]
Thanks for the PR. I left a few comments.

File: ui/component/searchScreenInput/view.jsx (hunk 1 above)
Comment (TomasGonzalez) [reply ui/component/searchScreenInput/view.jsx:26]: Hello @seanyesmunt, I have fixed all the request's for this PR except for this one, the reason why I did it this way in the first place is that electron has a weird behavior when using the `webContents.findInPage` method. Basically every time it is called, the input loses focus and I have been unable to fix this on my end. I really would like to finish implementing this feature, so If you have any advice I will take it, or I can implement it as I previously did if you allow me.

Comment (TomasGonzalez) [reply ui/component/searchScreenInput/view.jsx]: I have re-uploaded the PR with all the other comments fixed, except for this. I gave it a try once again and I was unable to get the desired behavior with the webContents.findInPage method.

Comment (TomasGonzalez) [reply ui/component/searchScreenInput/view.jsx]: @seanyesmunt Am I allowed to install external libraries? found one that might be of help with the misbehavior that I'm facing.

Comment (tzarebczan) [reply ui/component/searchScreenInput/view.jsx]: I think that's fine. Also, check out some of the comments in https://github.com/lbryio/lbry-desktop/issues/5877 for other feedback.

Comment (TomasGonzalez) [reply ui/component/searchScreenInput/view.jsx]: > I think that's fine. Also, check out some of the comments in #5877 for other feedback.

Mhh... Interesting seems that we are facing the same issues.

**Comment by kauffj** [comment]
@TomasGonzalez are you coming back to finish this?

**Comment by TomasGonzalez** [comment]
> @TomasGonzalez are you coming back to finish this?

Yes! I have been working on it on the weekends, have been posting my blockers and progress in https://github.com/lbryio/lbry-desktop/issues/5877. Can only work on it on the weekends so might take a while...

**Pull request closed without merging**